CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Removed
+- Removed support for Conjur v4 and the `CONJUR_VERSION` env variable
```

Comment:
```suggestion
### Removed
- Removed support for Conjur v4 and the `CONJUR_VERSION` env variable
  [cyberark/conjur-authn-k8s-client#505](https://github.com/cyberark/conjur-authn-k8s-client/pull/505)
```

Reply:
Oh, also, this needs to be under a version header!

It looks like 0.24.0 is the latest released version, so I think the 0.24.1 and 0.24.2 content would need to be included under whatever version we use for the v4 support drop.